Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/GoogleAPIVersion.kt

```diff
@@ -22,11 +22,15 @@ class GoogleAPIVersion @Inject constructor() {
             val currentVersion = InternalExposureNotificationClient.getVersion()
             currentVersion >= compareVersion
         } catch (apiException: ApiException) {
-            if (apiException.statusCode == CommonStatusCodes.API_NOT_CONNECTED) false
-            else throw apiException
+            isGoogleAPIConnected(apiException)
```

Comment:
Extracting the function here makes it less readable, because it's unexpected that it's a function that throws an exception.

Something like this would be nicer:

```kotlin
        if (apiException.statusCode == CommonStatusCodes.API_NOT_CONNECTED) {
            throw apiException
        }
        return false
```

Reply:
implemented